### ISSUE TITLE
How to manually add a server for Nord VPN

### ISSUE THREAD

**Issue by SethL** [qdm12/gluetun-wiki#17]
I am trying to connect to uk2504.nordvpn.com.  I can connect to it via the mac desktop client and everything works fine.  When I create the SERVER_HOSTNAMES variable in portainer and add  uk2504.nordvpn.com to it, I get an error and a list off all of the servers that Gluetun says I can connect to. However 2504 is not on that list.  

I went down the road of editing the servers.json list to include that server.  When I execute the container I get the same error msg but I also see where it says it is merging the lists together so it should have picked it up right?  I also tried running the docker command to update servers.json but that still doesnt include the server I want to connect to. 

TLDR: I want to specify a wireguard server for Nord that does not exist in the servers.json.  Is this possible?

**Comment by dozer1dave** [comment]
Same issue but with server us10179.nordvpn.com

**Comment by walmer26** [comment]
I am not sure. But to shed some light. Go to [this section](https://github.com/qdm12/gluetun-wiki/blob/main/setup/options/wireguard.md). It will tell you how to add a Wireguard conf file and bind it to the Gluetun container volume and what environment variable to use under "environment:" section of your compose file. I think it will help you if the hostname is not in the raw list and you just want to add a custom one.

Also, try to add an updater to your setup, can be found [here](https://github.com/qdm12/gluetun-wiki/blob/main/setup/options/updater.md).

Regards.

**Comment by kahliasmith** [comment]
Likewise, but with the OpenVPN protocol and ch416.nordvpn.com

**Comment by qdm12** [comment]
> I want to specify a wireguard server for Nord that does not exist in the servers.json. Is this possible?

You can by bumping the servers.json nordvpn version (for example from 1 to 2). The merging will detect this and use the bigger version, whatever the update time is.

It's not really documented on purpose, because:
- the update code should be updated to get these servers, instead of manually adding servers
- this could (unlikely due to timestamps but still) break with a future Gluetun program, where the version is bumped because the server model is changed

The servers data is fetched from https://api.nordvpn.com/v1/servers/recommendations?limit=0 I'm not sure why `us10179.nordvpn.com` isn't there, maybe worth reaching out to NordVPN's support about this?
